open_lm/main.py

```diff
@@ -416,12 +434,29 @@ def main(args):
 
     if args.distributed:
         if args.fsdp:
+            transformer_layer_cls = None
+
+            if args.hf_model is not None:
+                # retrive the user specified block class for fsdp
+                for _, target_cls in model.named_modules():
+                    if args.hf_fsdp_block in type(target_cls).__name__:
+                        transformer_layer_cls = {
+                            type(target_cls),
+                        }
+                        break
+
+                if transformer_layer_cls is None:
+                    print(f"--hf-fsdp-block {args.hf_fsdp_block} not found in --hf-model {args.hf_model}")
+                    return -1
+
+            else:
+                transformer_layer_cls = {
+                    Block,
```

Comment:
nit: can you remove the comma here and on line 444 so Black formats it into one line?